toolkits/global/packages/global-ratings-survey/README.md

```diff
@@ -0,0 +1,134 @@
+# Global Ratings Survey
+
+```text
+Component README follows Elements’ template, covering:
+- [Description]
+- [Demo]
+- When to use and when not to use this component
+- How it works
+- Installation
+- Template
+- Research on this component
+
+## Description
+- Designed to be used at multiple points in a journey
+- Collects information in a passive way
+- Designed to be comparable for data and collect any instances of major errors on our websites continuously
+- Designed to be unobtrusive - can be placed any page at any point in a user journey
+## When to use this component
+- Use this component for the CSAT method
+- Used in combination with other survey tools
+  - Could be used in combination with an additional survey at the end of a task / exit survey
+## When not to use this component
+- Do not use this component for NPS. NPS  is a different use-case - not an either or. If you need to use NPS - it should be placed at the end / exit of a task.
+## How to use (setup)
+Who to contact
+The question can be not changed
```

Comment:
@benjclark Are these your own notes?

Reply:
Yep, looking at the rendered file I don't think any of this text should be appearing?

Reply:
Sorry, I have removed this now. They were there as James A and Niamh had made TODO notes on the documentation and this seemed like the best place to store them at the time. I have now created a documentation site for this overall solution: https://customer-satisfaction-survey.public.springernature.app/docs/introduction and so I will store the TODOs as GitHub issues against that repo instead